debian/changelog

```diff
@@ -1,3 +1,9 @@
+kano-jupyter (1.0-1) unstable; urgency=low
```

Comment:
Doesn't need a new version as we've not ever released this, just merge this with the previous entry. Also, the version should match the next OS version to release so change this file to just be 

```
kano-jupyter (4.3.0-0) unstable; urgency=low

  * Added initial PoC integration on Kano OS
  * Added Python2 Kernel and a sample Python Opencv notebook

 -- Team Kano <dev@kano.me>  Thu, 14 Mar 2019 16:09:00 +0100
```

Reply:
Good point, rectified